### PR TITLE
iFixit Bundle: import and rename

### PR DESCRIPTION
**Pull request by danielbeardsley**
We previously symlinked this bundle into this location during an install step.

Now we've moved this files into this repo as they no longer depend on the other repo.
I didn't attempt to keep the history as some of these integrations are practically brand
new anyway and have very little interesting history.

We've added the one external dependency (guzzle) and included the composer.lock.

Also, we've included the patched `app.php` and `AppKernel.php` files which were
symlinked from the other repo as well.

See https://github.com/iFixit/ifixit/issues/34252

### CODE REVIEW

File: web/app_dev.php [hunk 1]
```diff
@@ -17,14 +17,20 @@
     exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
 }
 
-$loader = require_once __DIR__.'/../app/bootstrap.php.cache';
+$loader = require_once dirname($_SERVER['SCRIPT_FILENAME']).'/../app/bootstrap.php.cache';
 Debug::enable();
 
-require_once __DIR__.'/../app/AppKernel.php';
+require_once dirname($_SERVER['SCRIPT_FILENAME']).'/opt/akeneo/app/AppKernel.php';
```
Comment (sterlinghirsh) [inline web/app_dev.php:23]: Is this right? This is the only one with /opt/akeneo

Comment (danielbeardsley) [reply web/app_dev.php:23]: It's likely wrong and I don't think we've ever used `app_dev.php`. These files will be scrapped completely when we upgrade akeneo, so I'm OK with not fixing it.

**Comment by sterlinghirsh** [comment]
CR :zap:

**Comment by danielbeardsley** [comment]
QA 👍 

I exercised:

* The api this bundle adds (products can still be cloned)
* The post-save handlers
* The dependencies
* The above when the other framework is inaccessible

**Pull request merged**